arcgis-ios-sdk-samples/Layers/Stretch renderer/README.md

```diff
@@ -1,18 +1,54 @@
 # Stretch renderer
 
-This sample demonstrates how to use stretch renderer on a raster layer.
+Use a stretch renderer to enhance the visual contrast of raster data for analysis.
 
-## How to use the sample
+![Default stretch renderer](stretch-renderer-1.png)
+![Edited stretch renderer ](stretch-renderer-2.png)
+
+
+## Use case
 
-Tap the `Edit Renderer` button in the toolbar to open the settings for the stretch renderer. As you change the stretch type, the min, and the max, the raster layer will update accordingly.
+An appropriate stretch renderer can enhance the contrast of raster imagery, allowing the user to control how their data is displayed for efficient imagery analysis.
 
-![](image1.png)
-![](image2.png)
+## How to use the sample
+
+Tap the toolbar button to change any of the following properties.
 
+* AGSStandard deviation - a linear stretch defined by the standard deviation of the pixel values
+* AGSMin-max - a linear stretch based on minimum and maximum pixel values
+* AGSPercent clip - a linear stretch between the defined percent clip minimum and percent clip maximum pixel values
 
 ## How it works
 
-The sample uses `AGSStretchRenderer` class to generate stretch renderers. The settings provided by the user are put in the initializer `init(stretchParameters:gammas:estimateStatistics:colorRamp:)` to get a new renderer and the renderer is then set on the raster.
+1. Create an `AGSRaster` from a raster file.
+2. Create an `AGSRasterLayer` from the `AGSRaster`.
+3. Add the layer to the map.
+4. Create an `AGSStretchRenderer`, specifying the `AGSStretchParameters` and other properties.
+5. Apply the `AGSStretchRenderer` to the raster layer.
+
+## Relevant API
+
+* AGSColorRamp
+* AGSMinMaxStretchParameters
+* AGSPercentClipStretchParameters
+* AGSRaster
+* AGSRasterLayer
+* AGSStandardDeviationStretchParameters
+* AGSStretchParameters
+* AGSStretchRenderer
+
+## Offline data
+
+This sample uses a [raster file](https://arcgisruntime.maps.arcgis.com/home/item.html?id=95392f99970d4a71bd25951beb34a508). It is downloaded from ArcGIS Online automatically.
+
+## About the data
+
+The raster used in this sample shows an area in the south of the Shasta-Trinity National Forest, California.
+
+## Additional information
 
+See [Stretch function](http://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/stretch-function.htm) in the *ArcMap* documentation for more information about the types of stretches that can be performed.
```

Comment:
```suggestion
See [Stretch function](https://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/stretch-function.htm) in the *ArcMap* documentation for more information about the types of stretches that can be performed.
```
See if it can be https.